nats-base-client/types.ts

```diff
@@ -389,7 +389,7 @@ export interface StreamAPI {
   delete(stream: string): Promise<boolean>;
   list(): Lister<StreamInfo>;
   deleteMessage(stream: string, seq: number, erase?: boolean): Promise<boolean>;
-  getMessage(stream: string, seq: number): Promise<StoredMsg>;
+  getMessage(stream: string, query: MsgRequest): Promise<StoredMsg>;
```

Comment:
could avoid breaking change with `getLastMessageForSubject()`? if no biggie then lgtm

Reply:
added a shim and deprecation message - the jestream apis in the javascript clients are `preview`